src/connect-zbt-1/getting-started-with-zigbee/zbt-1-migrate-existing-zha-network-from-backup.md

```diff
@@ -67,32 +68,22 @@ Please note that not all settings can be migrated in this case:
 {% image "/static/img/connect-zbt-1/z2m-mqtt-dereg-device-01.png" "Removing Zigbee2MQTT devices (option 2)" %}
 {% stepContent %}
 
-1. If you don't want to delete the Mosquito Broker integration, you can remove the devices from the MQTT network.
-2. In the integration card, select **Devices**, under **Device info**, select **Delete**.
-3. Repeat this for every device.
-
-{% endstepContent %}
-{% endstep %}
-
-{% step "Plugging the extension cable into Connect&nbsp;ZBT-1" %}
-{% image "/static/img/connect-zbt-1/connect-zbt-1-raspi-01.jpg" "Plugging the extension cable into Connect&nbsp;ZBT-1" %}
-{% stepContent %}
-
-{% include "fragments/zbt-1-notice-interference.md" %}
+If you don't want to delete the **MQTT** integration, you can remove the devices from the MQTT network.
 
-1. Plug the Home Assistant Connect&nbsp;ZBT-1 into the USB extension cable.
-2. Plug the extension cable into your Home Assistant instance.
-3. If you are using the standalone [Home Assistant Container](https://www.home-assistant.io/docs/glossary/#home-assistant-container) installation method, make sure Connect&nbsp;ZBT-1 is [mapped to the container](https://www.home-assistant.io/installation/linux#exposing-devices).
+1. In the integration card, select **Devices**, under **Device info**, select **Delete**.
+2. Repeat this for every device.
```

Comment:
Since this assumes the user uses MQTT for non-Zigbee2MQTT devices, maybe mention that?

```suggestion
2. Repeat this for every Zigbee device.
```